pkg/cmd/system/info.go

```diff
@@ -153,13 +153,45 @@ func prettyPrintInfoDockerCompat(stdout io.Writer, stderr io.Writer, info *docke
 	// Storage Driver is not really Server concept for nerdctl, but mimics `docker info` output
 	fmt.Fprintf(w, " Storage Driver: %s\n", info.Driver)
 	fmt.Fprintf(w, " Logging Driver: %s\n", info.LoggingDriver)
-	fmt.Fprintf(w, " Cgroup Driver: %s\n", info.CgroupDriver)
-	fmt.Fprintf(w, " Cgroup Version: %s\n", info.CgroupVersion)
+	printF(w, " Cgroup Driver:  ", info.CgroupDriver)
```

Comment:
Extra space?

there isn't anything particular wrong with this but it might be simpler to either to print blank or wrap it like `info.warnings` below:

```
if info.CgroupDriver != "" { fmt.Fprintf(w, " Cgroup Driver: %s\n",, CgroupDriver ) }
```